pages/ksphere/kommander/1.2/tutorials/use-kcl-from-kubectl/index.md

```diff
@@ -149,7 +149,7 @@ metadata:
 spec:
   cluster:
     kubernetes:
-      version: 1.26.4
+      version: 1.18.0
```

Comment:
1.18.10 would be more exact

Reply:
That would be the default k8s 1.18 version that would be deployed with konvoy 1.6

Reply:
Thanks, sounds good. I'll have to retry it with that version number

Reply:
Works and looks good 👍